bundle/jsky.app.ot/src/main/scala/jsky/app/ot/gemini/editor/targetComponent/details/ConicNameEditor.scala

```diff
@@ -41,6 +41,10 @@ final class ConicNameEditor(date: HorizonsIO[Date]) extends JPanel with Telescop
       override def textBoxKeyPress(tbwe: TextBoxWidget): Unit =
         nonreentrant {
           spt.getTarget.setName(tbwe.getValue)
+          // editing the name so invalidate the horizons id.
```

Comment:
My understanding is that it is about avoiding firing events while in the middle of handling events.  It serves as a nice way of marking code that should be executed with event handlers off as opposed to explicitly keeping up with event registrations and removing them and then re-adding them at the end.